data/test/vtgate/schema_test.json

```diff
@@ -82,6 +82,12 @@
     "main": {
       "tables": {
         "main1": {},
+        "main2": {
```

Comment:
Not sure if we should take this in this review, but i think we should rename main,main1 and main2 to something better like the table names we have for data shards